submissions/zhenyakornilov/js-dom/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" id="root">
```

Comment:
It is a common practice to divide responsibility - ids for JS, classes for styling. Better to use the wrapper in this case and set styles for `wrapper`, not for `html`tag. Using html styling is even bad for performance reasons

Reply:
can I use :root pseudo-class here then?

Reply:
the common use of :root is for variables declaration. For such files root will be always `html`
https://developer.mozilla.org/en-US/docs/Web/CSS/:root